apps/web/components/user-feedback/feedback-table-filter-dropdown.tsx

```diff
@@ -0,0 +1,80 @@
+"use client";
+
+import {
+  DropdownMenu,
+  DropdownMenuCheckboxItem,
+  DropdownMenuContent,
+  DropdownMenuTrigger,
+} from "@repo/ui/components/dropdown-menu";
+import { Button } from "@repo/ui/components/button";
+import { useState } from "react";
+import { Filter } from "lucide-react";
+import { useRouter, useSearchParams } from "next/navigation";
+import { FEEDBACK_FILTERS } from "@/constants/constants";
+
+export function FeedbackTableFilterDropdown() {
+  const router = useRouter();
+  const searchParams = useSearchParams();
+
+  const filtersOnURLQuery = searchParams.get("sentiment")?.split(",") || [];
+  const [selectedFilters, setSelectedFilters] = useState<string[]>(
+    filtersOnURLQuery.filter((f) => FEEDBACK_FILTERS.includes(f)),
+  );
+
+  const handleToggleFilter = (filter: string, checked: boolean) => {
+    let newFilters = [];
+    if (checked) {
+      newFilters = [...selectedFilters, filter];
+    } else {
+      newFilters = selectedFilters.filter((f) => f !== filter);
+    }
+
+    setSelectedFilters(newFilters);
+
+    const params = new URLSearchParams(searchParams);
+    if (newFilters.length > 0) {
+      params.set("sentiment", newFilters.join(","));
```

Comment:
Could we extract "sentiment"  into a constant e.g. `const SENTIMENT_QUERY_PARAM_VALUE`  to be able change it or remove it quickly if needed. This also will help us track the usages.